docs/nightly/en/getting-started/quick-start.md

```diff
@@ -0,0 +1,370 @@
+('2024-07-11 20:00:14', 'host1', '/api/v1/resource', 'Error: Timeout'),
+('2024-07-11 20:00:15', 'host1', '/api/v1/resource', 'Error: Disk full'),
+('2024-07-11 20:00:16', 'host1', '/api/v1/resource', 'Error: Network issue');
+```
+
+## Query data
+
+### Filter by tags and time index
+
+You can filter data using the WHERE clause.
+For example, to query the latency of `host1` after `2024-07-11 20:00:15`:
+
+```sql
+SELECT * FROM grpc_metrics WHERE host = 'host1' AND ts > '2024-07-11 20:00:15';
+```
+
+```sql
++---------------------+-------+-------------+---------+
+| ts                  | host  | method_name | latency |
++---------------------+-------+-------------+---------+
+| 2024-07-11 20:00:16 | host1 | GetUser     |    3000 |
+| 2024-07-11 20:00:17 | host1 | GetUser     |     320 |
+| 2024-07-11 20:00:18 | host1 | GetUser     |    3500 |
+| 2024-07-11 20:00:19 | host1 | GetUser     |     100 |
+| 2024-07-11 20:00:20 | host1 | GetUser     |    2500 |
++---------------------+-------+-------------+---------+
+5 rows in set (0.01 sec)
+```
+
+You can also use functions when filtering the data.
+For example, you can use `approx_percentile_cont` to calculate the 95th percentile of the latency grouped by the host:
+
+```sql
+SELECT approx_percentile_cont(latency, 0.95) AS p95_latency, host FROM grpc_metrics WHERE ts >= '2024-07-11 20:00:10' GROUP BY host;
+```
+
+```sql
++-------------------+-------+
+| p95_latency       | host  |
++-------------------+-------+
+| 4164.999999999999 | host1 |
+|               115 | host2 |
++-------------------+-------+
+2 rows in set (0.02 sec)
+```
+
+### Range query
+
+You can use [range queries](/reference/sql/range#range-query) to monitor latencies in real-time.
+For example, to calculate the p95 latency of requests using a 5-second window:
+
+```sql
+SELECT 
+  ts, 
+  host, 
+  approx_percentile_cont(latency, 0.95) RANGE '5s' AS p95_latency 
+FROM 
+  grpc_metrics
+ALIGN '5s' FILL PREV;
+```
+
+```sql
++---------------------+-------+-------------+
+| ts                  | host  | p95_latency |
++---------------------+-------+-------------+
+| 2024-07-11 20:00:05 | host2 |         114 |
+| 2024-07-11 20:00:10 | host2 |         111 |
+| 2024-07-11 20:00:15 | host2 |         115 |
+| 2024-07-11 20:00:20 | host2 |          95 |
+| 2024-07-11 20:00:05 | host1 |       104.5 |
+| 2024-07-11 20:00:10 | host1 |        4200 |
+| 2024-07-11 20:00:15 | host1 |        3500 |
+| 2024-07-11 20:00:20 | host1 |        2500 |
++---------------------+-------+-------------+
+8 rows in set (0.15 sec)
+```
+
+### Full-text search
+
+The latency analysis above indicates that `host1` may be experiencing some issues.
+You can utilize the full-text search `matches(path, 'api/v1')` to identify error logs associated with the specific API `api/v1`:
+
+```sql
+SELECT 
+  ts, 
+  count(error) RANGE '5s' AS num_errors,
+  first_value(error) RANGE '5s' AS first_error, 
+  last_value(error) RANGE '5s' AS last_error,
```

Comment:
This doesn't look like a real use case. Who would care about "first log in every 5 seconds"?

Reply:
I'd suggest not applying `RANGE` on log table but metric table. For log table simply filtering should be enough. Aggregation on log text doesn't make sense.